tests/sample/sample.vcxproj

```diff
@@ -141,16 +141,6 @@
       <Outputs Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">$(OutputPath)%(Filename).o</Outputs>
       <Outputs Condition="'$(Configuration)|$(Platform)'=='Release|x64'">$(OutputPath)%(Filename).o</Outputs>
     </CustomBuild>
-    <CustomBuild Include="encap_reflect_packet.c">
```

Comment:
Why is this one removed?  Seems unrelated to moving schedule cleanup from program to object.

Reply:
The project fails to load.